community/modules/compute/schedmd-slurm-gcp-v6-nodeset/main.tf

```diff
@@ -42,7 +42,7 @@ locals {
   public_access_config = var.enable_public_ips ? [{ nat_ip = null, network_tier = null }] : []
   access_config        = length(var.access_config) == 0 ? local.public_access_config : var.access_config
 
-  service_account_email = coalesce(var.service_account_email, data.google_compute_default_service_account.default.email)
+  service_account_email = var.service_account_email
```

Comment:
Remove this local var, use `var.service_account_email` directly

Reply:
Done